spec/send_spec.js

```diff
@@ -114,8 +106,6 @@ describe('send', function() {
         monitoring.showHideList();
 
         authoring.sendTo('Politic Desk');
-        //Spell check confirmation modal save action
-        authoring.confirmSendTo();
```

Comment:
I think that is better to add back on your PR and in this way it will be validated that the issue is fixed.